### PR TITLE
Minor lint tweaks

### PR DESCRIPTION
**Pull request by XhmikosR**
* use `String.startsWith`
* use `String.includes`

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ const gulpSass = (options, sync) => transfob((file, enc, cb) => { // eslint-disa
     return cb(new PluginError(PLUGIN_NAME, 'Streaming not supported'));
   }
 
-  if (path.basename(file.path).indexOf('_') === 0) {
+  if (path.basename(file.path).startsWith('_')) {
```
Comment (XhmikosR) [inline index.js:24]: `file.path` cannot be `null`/`undefined` here AFAICT so this should be fine

**Comment by xzyfer** [comment]
Thanks

**Comment by XhmikosR** [comment]
Will you cut a new version too?

**Comment by xzyfer** [comment]
Yes I will tomorrow when I'm on a development machine.

**Pull request merged**